blocks/library/freeform/freeform-block.js

```diff
@@ -405,6 +405,7 @@ export default class FreeformBlock extends Component {
 			</BlockControls>,
 			<TinyMCE
 				key="editor"
+				className={ className }
```

Comment:
What alternative do we have? adding a wrapper? we want to add the className to the root element of the block.